examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -102,6 +102,27 @@ typedef Particles<ParticleDescription<
 
 /*--------------------------- ions -------------------------------------------*/
 
+/*! Specify (chemical) element
+ * 
+ * Proton and neutron numbers define the chemical element that the ion species
+ * is based on. This value can be non-integer for physical models taking  
+ * charge shielding effects into account. @see http://en.wikipedia.org/wiki/Effective_nuclear_charge
+ * 
+ * It is wrapped into a struct because of C++ restricting floats from being 
+ * template arguments. 
+ * 
+ * Do not forget to set the correct mass of the atom in @see physicalConstants.param !
```

Comment:
I think we can write a full-flavored atom-wrapper directly here (and remove the mass/charge crap from `physicalConstants`)